tests/data/validate/validate.test.ts

```diff
@@ -0,0 +1,77 @@
+import Ajv from 'ajv';
```

Comment:
Should you move the tests to the file below? This file is in the data directory, which is a bit strange.

Reply:
schema-validate.test.ts file or to create a sub directory of schema within data? This folder structure is kinda jank for sure.

Reply:
All good, don't mind this comment, will resolve it.